autoload/lsp/ui/vim/references.vim

```diff
@@ -221,5 +221,5 @@ function! lsp#ui#vim#references#jump(offset) abort
 
     " Jump
     let l:target = w:lsp_reference_positions[l:index][0:1]
-    silent exec 'normal! ' . l:target[0] . 'G' . l:target[1] . '|'
```

Comment:
@prabirshrestha How about to use setpos() or cursor() ?

Reply:
That would be better. Not much fan of exec.

Reply:
Pushed a commit changing that line to use cursor() instead.